### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by fernando-silvabr66**
Add classroom and book classes
Add rental class to follow associations requirements
Update person and student classes

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -14,6 +14,8 @@ def initialize(age, name = 'Unknown', parent_permision: true)
 
     @age = age
 
+    @rentals = []
+
     @parent_permision = parent_permision
   end
```
Comment (omarsalem7) [inline person.rb:21]: - [x] Kindly add getter and setter to rentals inside Person class by adding it to `attr_accessor`

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @fernando-silvabr66 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (fernando-silvabr66) [reply person.rb:21]: Thanks for the suggestion.

**Review by gemmen29 — APPROVED** [review]
Hi @fernando-silvabr66, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._00

**Pull request merged**